go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/viper v1.6.2
 	github.com/streadway/quantile v0.0.0-20150917103942-b0c588724d25 // indirect
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
Updated to include this fix stretchr/testify#808 to prevent potential intermittent test failures

Reply:
Can you quickly split this in a separate PR?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/876